static/metricsgraphics/css/font-awesome.min.css

```diff
@@ -0,0 +1,4 @@
+/*!
+ *  Font Awesome 4.2.0 by @davegandy - http://fontawesome.io - @fontawesome
```

Comment:
Font-awesome 4.2.0 was also released in 2014.

Reply:
I'd need to pull in a new version and test it to see if it works. I'm simply using the version here that I've always been using since I coded the original Bugzilla Socorro Lens extension.